### ISSUE TITLE
mypy pyi-file is required.

### ISSUE THREAD

**Issue by Klayflash** [lycantropos/cfractions#2]
I've found that the library contains no pyi-file.
Is it will be added to mypy repository that described in [Stub files](https://mypy.readthedocs.io/en/latest/stubs.html)?
I think a template can be obtained from official mypy repository [fractions.pyi](https://github.com/python/typeshed/blob/master/stdlib/fractions.pyi)

**Comment by lycantropos** [comment]
Why do we need this? Current package already has type annotations for pure-Python version
https://github.com/lycantropos/cfractions/blob/22ede26a22786f5480174e2306414ff38f7c7b77/cfractions/__init__.py#L22-L349
and PyCharm for example is able to process them.

**Comment by Klayflash** [comment]
Steps to reproduce:

1. install mypy: `python -m pip install mypy`
2. install cfractions: `python -m pip install cfractions==1.4.0`
3. run `"C:\Users\USER_NAME\AppData\Local\Programs\Python\Python39\Scripts\dmypy" run --timeout 7200 -- --no-color-output --namespace-packages "D:\src\test2.py"`

output:
```
D:\src\test2.py:1: error: Skipping analyzing "cfractions": found module but no type hints or library stubs
D:\src\test2.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
```
test2.py content is:
```
from cfractions import Fraction

number = Fraction(12)
```

**Comment by Klayflash** [comment]
If change `from cfractions import Fraction` to `from fractions import Fraction` then no issues.

**Comment by lycantropos** [comment]
Fixed by adding interface files into distribution, available from version `2.0.0`
